docs/AAD-OAuth.md

```diff
@@ -322,18 +325,157 @@ curl -v -H "Authorization: Bearer $acr_access_token" https://$registry/v2/_catal
 ```
 Note that `curl` by default does the request as a `GET` unless you specify a different verb with the `-X` modifier.
 
-This should result in a status 200 OK, and a body with a JSON payload listing the repositories held in this registry:
+This should result in a status 200 OK, and a body with a JSON payload listing the repositories held in this registry, using `...` to shorten it for illustrative purposes:
+
+```json
+{"repositories":["alpine","contoso-marketing","hello-world","node",...]}
```

Comment:
Can you avoid the `...` and use the original example since `...` results in an invalid JSON object